src/components/Quiz/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { useContext } from "react";
+import "../../scss/Quiz.scss";
+import Answers from "./Answers";
+import { QuizContainer } from "../../QuizContainer";
+import YouTube from 'react-youtube';
+import Header from "./Header";
+
+const Quiz = () => {
+
+  const {question, questionNumber, youTubeId} = useContext(QuizContainer.QuizContext);
+  const hideQuestionPanel = questionNumber === 0 || question.status === "hidden";
+  const {seconds_left: secondsLeft, title} = question;
+
+  const videoOptions = {
```

Comment:
You can wrap it in useMemo or extract outside the Quiz component to skip redeclaration